src/windows.c

```diff
@@ -53,11 +54,11 @@ static void wait_worker_threads(struct pthreadpool* threadpool, uint32_t event_i
 }
 
 static uint32_t wait_for_new_command(
-	struct pthreadpool* threadpool,
+	struct thread_info* thread,
```

Comment:
Why the API change?

Reply:
Because now wakeup and command both are per thread.